### PR TITLE
Add version number option to pinter

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**

### CODE REVIEW

**Comment by JakeRoggenbuck** [comment]
[6b5d](https://github.com/plancklang/pinter/pull/8/commits/6b5d99a24a0ea9f326f8b22f2e1ae5b773995c41) fixed the issue with [6f4e](https://github.com/plancklang/pinter/pull/8/commits/6f4eac36a4cb4f1f379251a9d2fc2d40e1609bc9)

File: src/util.h [hunk 1]
```diff
@@ -12,6 +12,8 @@
 #define PL_SUCCESS 0
 #define PL_ERROR 1
 
+#define VERSION "0.1"
```
Comment (adamhutchings) [inline src/util.h:15]: Could you define VERSION as `0.1` and not. `"0.1"` ?

File: src/main.c [hunk 2]
```diff
@@ -11,10 +11,15 @@ void usage() {
 	printf("\
 Usage:\n\
 	--help see this message\n\
+	-v see the version\n\
 	[file]: execute [file]\n\
 ");
 }
 
+void info() {
+	printf("Pinter %s\n", VERSION);
```
Comment (adamhutchings) [inline src/main.c:20]: Change to `%f`, I believe. Make sure to test.

**Review by adamhutchings — CHANGES_REQUESTED** [review]
It's better if VERSION is a float.

**Comment by JakeRoggenbuck** [comment]
If it’s a float and we printf as %f, it will print 0.1000000 and I don’t know how to change that. I agree we should have the value something more space efficient. Can we print a float as a string?

**Pull request merged**